analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -449,6 +449,19 @@ open class Npm(
         return jsonMapper.readTree(process.stdout)
     }
 
+    protected open fun findWorkspaceSubmodules(moduleDir: File): List<File> {
```

Comment:
Since the method is now *protected*, it becomes part of the API of the class and should therefore be documented - especially if it is designed for extension. It would be helpful to describe when this function is called as part of the resolving processes and how its results are further processed.